### ISSUE TITLE
Is it possible to support Crap patrol 2 ?

### ISSUE THREAD

**Issue by Evonos** [DarthVan/StarCraft2-Bank-Restorer#8]
I want to move my bank file from Europe to USA and back  ( Eu player that wants to play at night at USA )

**Comment by DarthVan** [comment]
Yes, it is possible if I have some free time to finish encrypting/decrypting this arcade. But I'm too busy with commercial projects right now.

**Comment by Evonos** [comment]
> Yes, it is possible if I have some free time to finish encrypting/decrypting this arcade. But I'm too busy with commercial projects right now.

okay thanks ! no pressure

**Comment by Evonos** [comment]
Do you think this can be done ?

**Comment by DarthVan** [comment]
> Do you think this can be done ?

Yep. some encrypt/decrypt code was ported from galaxy to typescript a few months ago, and i forgot about that map for a while 😅
I still need to test it and make ui for the map. I can't give any timeframes yet..

But you can help a little by attaching here some bank file with good progress. It will help with testing.
